### DISCUSSION TITLE
Add a Clicker function for the specified window coordinates, or search for an item by OTP or via base64 code of the graphic item.

### DISCUSSION THREAD

**Discussion by Korvin2021** [AutomaApp/automa#737] [Ideas]
Add a Clicker function for the specified window coordinates, or search for an item by OTP or via base64 code of the graphic item.

Often clicker activation is required, and having a selector is not available or does not work in the additional window. 
It would also be nice to add a simulation of mouse cursor inactivity, this function is often needed.

**Comment by nick0zer** [comment]
Finding an image function as well would be useful. Not sure if it's already in there.
